index.js

```diff
@@ -29,58 +30,51 @@ const availableCommandsMessage = `Available commands are:-
  */
 module.exports = app => {
 	/* --------------------------------- ANCHOR Automation --------------------------------- */
-	app.on('pull_request.opened', addReadyForReviewLabel);
+	app.on("pull_request.opened", addReadyForReviewLabel);
 
-	app.on('pull_request_review.submitted', addApprovedLabel);
-	app.on('pull_request_review.submitted', changesRequestLabel);
-
-	app.on('pull_request.closed', addMergedLabel);
-
-	app.on(
-		['pull_request.edited', 'pull_request.labeled'],
-		pullRequestWIPLabelAutomation
-	);
-
-	app.on('issues.closed', addLabelToIssueOnClose);
+	app.on("pull_request_review.submitted", addApprovedLabel);
+	app.on("pull_request_review.submitted", changesRequestLabel);
+	app.on("pull_request.closed", addMergedLabel);
+	app.on("issues.closed", addLabelToIssueOnClose);
```

Comment:
Codacy has a fix for the issue: [Replace `↹` with `··`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
  app.on("issues.closed", addLabelToIssueOnClose);
```